### ISSUE TITLE
PieChart Warning: Each child in a list should have a unique "key" prop.

### ISSUE THREAD

**Issue by radekzz** [Abhinandan-Kushwaha/react-native-gifted-charts#302]
There is an error in default settings
```
Warning: Each child in a list should have a unique "key" prop.
Check the render method of `PieChartMain`. See https://reactjs.org/link/warning-keys for more information. in PieChartMain (at PieChart/index.tsx:138)
```

Somehow i cannot create a PR, so here is the fix:

replace
```
return (
              <>
                {/* <Line x1={mx} x2={cx} y1={my} y2={cy} stroke="black" /> */}
                {showTextBackground && (
                  <Circle
                  key={index + 'b'}
```
with
```
return (
              <React.Fragment key={index + 'b'}>
                {/* <Line x1={mx} x2={cx} y1={my} y2={cy} stroke="black" /> */}
                {showTextBackground && (
                  <Circle
```

But actually i found lots of TS errors in `src/PieChart/main.tsx`

**Comment by Abhinandan-Kushwaha** [comment]
Hi @radekzz Thanks for this suggestion. I will soon update the PieChart file.

**Comment by Abhinandan-Kushwaha** [comment]
The warning due to the absence of `key` prop is now fixed in version `1.3.0`🎉
